remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java

```diff
@@ -374,15 +374,15 @@ public RemotingCommand invokeSync(String addr, final RemotingCommand request, lo
                 doAfterRpcHooks(RemotingHelper.parseChannelRemoteAddr(channel), request, response);
                 return response;
             } catch (RemotingSendRequestException e) {
-                log.warn("invokeSync: send request exception, so close the channel[{}]", addr);
+                log.warn("invokeSync: send request exception, so close the channel[{}]", channel);
```

Comment:
I think `RemotingUtil.socketAddress2String(channel.remoteAddress())` would be better